crates/turbopack/src/module_options/module_options_context.rs

```diff
@@ -239,6 +242,9 @@ pub struct ModuleOptionsContext {
     pub placeholder_for_future_extensions: (),
     #[serde(default)]
     pub enable_tree_shaking: bool,
+
+    #[serde(default)]
+    pub compile_time_info: Option<CompileTimeInfoVc>,
```

Comment:
@sokra Is `Option<CompileTimeInfoVc>` fine here?

Reply:
As this type derives `Default`, I used `Option` but I'm not sure

Reply:
I guess yes

Reply:
If so, is it fine to use `Option<CompileTimeInfoVc>` for `EcmascriptOptions`?